.github/workflows/shellspec.yaml

```diff
@@ -11,4 +11,9 @@ jobs:
         go-version: '1.24'
     - run: |
         make shellspec
-      name: Run shell/bash script unit tests with shellspec
\ No newline at end of file
+      name: Run shell/bash script unit tests with shellspec
+    - run: echo "Shell/Bash script unit tests failed. Please check the logs for details. To run the shellspec unit tests locally, please refer to ./spec/README.md"
+      if: failure()
+      name: Print failure message
+    - run: echo "Shell/Bash script unit tests completed successfully!"
```

Comment:
I think this will run even in the failure case

Reply:
Actually it won't. I tested with both cases.

Reply:
Right it won't be printed.
`This is because GitHub Actions stops executing subsequent steps in a job by default when a step fails, unless you explicitly configure it to continue.`

Reply:
sorry, I was referring to the `Print completion message` case, though I'm assumign you also double-chceked this

Reply:
right I mean the completion message too 🙂

Reply:
ah I see, what you pasted above probably applies to _all_ steps in the job rather than just the immediately proceeding one